File: pl-domain/doctor-service/src/main/java/com/polylab/doctors/DoctorServiceApplication.java
package com.polylab.doctors;

import com.jxareas.jxcore.annotations.EnableJxCore;
import com.jxareas.jxcore.annotations.EnableNonRestrictiveCorsPolicy;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@EnableNonRestrictiveCorsPolicy
@EnableJxCore
@SpringBootApplication
public class DoctorServiceApplication {

    public static void main(String[] args) {
        SpringApplication.run(DoctorServiceApplication.class, args);
    }

}

File: pl-domain/doctor-service/src/main/java/com/polylab/doctors/DoctorServiceApplication.kt
package com.polylab.doctors;

import com.jxareas.efficax.annotations.EnableEfficax
import com.jxareas.efficax.annotations.EnableNonRestrictiveCorsPolicy
import com.jxareas.efficax.extensions.runApp
import org.springframework.boot.autoconfigure.SpringBootApplication

@EnableNonRestrictiveCorsPolicy
@EnableEfficax
@SpringBootApplication
open class DoctorServiceApplication

fun main(vararg args: String) = runApp(DoctorServiceApplication::class, *args)
